packages/prerender-fargate/lib/prerender/server.js

```diff
@@ -12,15 +12,50 @@
 'use strict';
 
 const prerender = require('prerender');
+const util = require('prerender/lib/util');
 const crypto = require('crypto');
 const s3Cache = require('prerender-aws-s3-cache');
+const logger = require('./utils/logger')
+
+/**
+ * Replace prerender's util log function with our own which uses pino to log
+ *
+ * @param  {...any} args 
+ * @returns 
+ */
+util.log = function (...args) {
+    if (process.env.DISABLE_LOGGING) {
+        return;
+    }
+
+    logger.info(args.join(' '));
+};
 
 const server = prerender({
     chromeFlags: ['--no-sandbox', '--headless', '--disable-gpu', '--disable-web-security', '--remote-debugging-port=9222', '--hide-scrollbars', '--disable-dev-shm-usage'],
     forwardHeaders: true,
     chromeLocation: '/usr/bin/chromium-browser'
 });
 
+server.use({
+    beforeSend: (req, res, next) => {
+        const ms = new Date().getTime() - req.prerender.start.getTime();
+
+        if (req.prerender.url !== "health") {
+            logger.render({
```

Comment:
It's not clear what this is doing? I assume it only renders the logs if the req.prerender.url is not health? What about process logs? Startup, shutdown, etc?

Reply:
Because this is a server plugin on `beforeSend` this `render` log happens always before the server sends a response. And that means ELB healthchecks too.

`logger.render` is a custom log level that we would like to use to log crawler and bot requests.

Everything else is logged with `logger.info` as you can see in the PR's description screenshot

> ![image](https://github.com/aligent/cdk-constructs/assets/40108018/009acf9e-d05a-49fb-9f6c-4d20f404ba6f)

Reply:
I've made a dedicated `/health` endpoint so that returns `200 OK` right away.

Reply:
Can you add a comment for that?